scripts/zones/Ghelsba_Outpost/mobs/Colo-colo.lua

```diff
@@ -3,13 +3,18 @@
 --  MOB: Colo-colo
 -- BCNM20
 -----------------------------------
+require("scripts/zones/Ghelsba_Outpost/MobIDs");
+-----------------------------------
 
 function onMobSpawn(mob)
 end;
 
 function onMobEngaged(mob,target)
-    GetMobByID(17350930):updateEnmity(target);
-    GetMobByID(17350931):updateEnmity(target);
+    for i = WINGS_OF_FURY_OFFSET, WINGS_OF_FURY_OFFSET + 2 do
```

Comment:
Code you are replacing looks like old code from before bcnm was properly setting Agro. We shouldn't need to update enmity at all.

Reply:
Even better.  Updated.